.github/workflows/closed-issue-message.yml

```diff
@@ -0,0 +1,17 @@
+name: Closed Issue Message
```

Comment:
nit: To be consistent with other files, use 2 spaces indentation

Reply:
Updated!